Automattic/woocommerce-payments
# Add date_between filter for Authorization Reporting API

Fixes #7445 

#### Changes proposed in this Pull Request
- Updated default value for `sort` filter
- Added `date_between` filter. The filter should be of type array, with the first value the 'from date', and the second value the 'to date'
- Updated docs for authorization API
- 








#### Testing instructions
- Create REST API Keys as described [here](https://woocommerce.github.io/woocommerce-rest-api-docs/#rest-api-keys)
- Sample curl URL for GET `reports/authorizations`
```
curl 'https://<host>/wp-json/wc/v3/payments/reports/authorizations?page=1&per_page=10&sort=created&direction=desc&user_timezone=+05:30&date_between[]=2022-10-10 09:06:50&date_between[]=2022-12-16 09:06:50' -u <ck_>:<cs_> -H "Content-Type: application/json"
```
- check if `date_between` filter works as expected. 

https://github.com/Automattic/woocommerce-payments/assets/82826872/958749b2-0ee7-4744-b95f-ae1a726bb709







-------------------


**Post merge**
